agent_identity/src/document/aggregate.rs

```diff
@@ -706,7 +705,7 @@ pub mod test_utils {
             .type_("LinkedDomains")
             .service_endpoint(
                 ServiceEndpoint::from_json_value(json!({
-                    "origins": [config().url],
+                    "origins": [config().public_url.clone()],
```

Comment:
This is just test code, but shouldn't this be equivalent to the production code:

```
config().public_url.origin().clone()
                    ^^^^^^^^
```

just to make sure?

Reply:
Agreed ==> fixed :ok_hand: